src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -121,8 +121,17 @@ public VisualStudioWorkspaceImpl(ExportProvider exportProvider, IAsyncServicePro
             FileWatchedReferenceFactory = exportProvider.GetExportedValue<FileWatchedPortableExecutableReferenceFactory>();
 
             FileWatchedReferenceFactory.ReferenceChanged += this.RefreshMetadataReferencesForFile;
+
+            ExternalErrorDiagnosticUpdateSource = new ExternalErrorDiagnosticUpdateSource(
```

Comment:
The context here is we generally always want to defer as late as possible, so that's just an automatic question you're going to get from me. We had perf problems with our package because it loaded too many things that weren't needed until later; I'm just making sure we're not re-creating that later.